### ISSUE TITLE
Password and users empty

### ISSUE THREAD

**Issue by pey33** [nilsteampassnet/TeamPass#3058]
### Steps to reproduce
1. update TeamPass 2.1.27.36 to 3.0.0.11
2.
3.

### Expected behaviour
When I click on the eye to display the password, it is empty 
I can log in with registered users but the list is empty from the administrator account.

### Actual behaviour
see password or copy password

### Server configuration
**Operating system**:

**Web server:**
apache2.4.52
![2022-02-28_00h39_48](https://user-images.githubusercontent.com/100541605/155904918-607ddc6c-d0ae-4cfb-9e85-b61f102a9830.png)
![2022-02-28_00h41_06](https://user-images.githubusercontent.com/100541605/155904963-653d9c37-6c00-4b1f-b44d-7fcf1895297f.png)

**Database:**
Mariadb 10.6.7
**PHP version:**
7.4
**Teampass version:** 
3.0.0.11
**Teampass configuration file:**

**Updated from an older Teampass or fresh install:**
PLEASE attach to this issue the file `/includes/config/tp.config.php`.

### Client configuration
**Browser:**
all
**Operating system:**
Debian 11
#### Web server error log
``` no error
Insert your webserver log here
```
no error
#### Log from the web-browser developer console (CTRL + SHIFT + i)
```
Insert the log here and especially the answer of the query that failed.
```

**Comment by RemiFlandrois** [comment]
I have the same issue and i resolved it by passing the language in english in the admin "My profile" menu and in "Settings => Options=>Default user interface language".
If you want you can perform a rebuild of the configuration file in "Action" menu.

**Comment by DaRoxor** [comment]
Did the trick but without effect. In the doubt I will switch the default language from french to english before to do the update .. keep you in touch asap.

**Comment by DaRoxor** [comment]
no change :)

**Comment by RemiFlandrois** [comment]
I have test this morning the 3.0.0.13 and have the same issue than you (and my change language trick no longer work)

**Comment by nilsteampassnet** [comment]
I'm not sure to understand the case.
This occurs after upgrading from 2.x?

**Comment by DaRoxor** [comment]
Yes exactly it's occur after upgrading from 2. Version. From fresh install there is no issue!

**Comment by nilsteampassnet** [comment]
@DaRoxor do you think you could share your screen with me?
Please contact me by Telegram `@nilsl `

**Comment by RemiFlandrois** [comment]
Hi @nilsteampassnet, if @DaRoxor is not available, i have a pre prod environnement and we could share a screen when you want ! (and speak french !)

**Comment by nilsteampassnet** [comment]
HI @RemiFlandrois 
With pleasure, can you contact me by Telegram

**Comment by gmontequi** [comment]
I think the problem is in the encryption.

The best way to avoid the error is to convert the user with which you have created the items in administrator from the user management console and then, in the upgrade wizard, put that user in the "teampass administrator" part. Then during the upgrade the encryption keys will be generated for that user and will be applied later to the items and the process will not fail. In my case at least, I had to, once the upgrade was done, enter the user administration console (with my user, not admin) and change the password for each one of them. That makes the items to be encrypted again with the keys and it will work without problems. At least this is the only solution I have found. Whenever I upgraded with the admin user I had the same error as you. Once I did the upgrade with my user (previously converting it to admin) it worked without problems.

Best regards,

**Comment by DaRoxor** [comment]
@nilsteampassnet 
I can't share my screen for security reason, sorry :( 

@gmontequi 
I can't do this trick because we have thousand items and hundred users :)

**Comment by gmontequi** [comment]
Try to create a new admin user and test the upgrade with that user.

best regards,

**Comment by DaveWebb2** [comment]
I too have this issue.  I upgraded from 2.1.27.36 to 3.0.0.11 and none of the users can see any passwords, they are all blank.  There were no errors reported during the upgrade.  I can't see any fresh errors in the apache error log and there are no new errors on the log page within the application.

Here are my details:

Ubuntu 22.04.4 LTS

PHP version: 8.2.16
Memory limit: 128M
Memory usage: 499.19 KB
Maximum time execution: 30
Maximum file size upload: 2M

Server version: 10.6.16-MariaDB-0ubuntu0.22.04.1
Database size: 44.56MB

**Comment by douglasarkadium** [comment]
Hi @DaveWebb2 and @nilsteampassnet,

Did you find a solution? I ask because I updated TeamPass from version 2.1.26 to 2.1.27.36 and then to 3.1.1. After making some adjustments, everything worked fine. However, when I set up LDAP, new users cannot see passwords; all passwords are blank. If I remove the LDAP configuration and add the user again manually, the user can see password. But, if I try to sync with LDAP again, it doesn’t work, the passwords “disappear”.

It’s important to mention that old users, even when synced with LDAP, work fine. The issue only happens with new users.

By the way, I am using local groups to define permissions, not the ones from Active Directory (AD).

**Comment by DaveWebb2** [comment]
> Hi @DaveWebb2 and @nilsteampassnet,
> 
> Did you find a solution? I ask because I updated TeamPass from version 2.1.26 to 2.1.27.36 and then to 3.1.1. After making some adjustments, everything worked fine. However, when I set up LDAP, new users cannot see passwords; all passwords are blank. If I remove the LDAP configuration and add the user again manually, the user can see password. But, if I try to sync with LDAP again, it doesn’t work, the passwords “disappear”.
> 
> It’s important to mention that old users, even when synced with LDAP, work fine. The issue only happens with new users.
> 
> By the way, I am using local groups to define permissions, not the ones from Active Directory (AD).

No, I am still without a solution after about 5 months of trying to upgrade.  I may have to look at moving to a different product, as I need to get away from this old version of TeamPass and the server it runs on.  There's clearly a bug somewhere that needs fixing.

**Comment by douglasarkadium** [comment]
Hi @nilsteampassnet 

I contacted you on Telegram. If you want to debug the problem, I'm available to demonstrate the scenario so we can work on finding a solution.

**Comment by nilsteampassnet** [comment]
Closed automatically due to inactivity (no response for over 1 year).
